video_xblock/tests/base.py

```diff
@@ -0,0 +1,53 @@
+"""
+Vide xblock base classes.
+"""
+import unittest
+import mock
+
+from xblock.field_data import DictFieldData
+from xblock.test.tools import TestRuntime
+
+from video_xblock import VideoXBlock
+
+
+class VideoXBlockTestBase(unittest.TestCase):
+    """
+    Base video_xblock test class.
+    """
+    def setUp(self):
+        """
+        Creates a XBlock VideoXBlock for testing purpose.
+        """
+        super(VideoXBlockTestBase, self).setUp()
+        runtime = TestRuntime()  # pylint: disable=abstract-class-instantiated
+        self.xblock = VideoXBlock(
+            runtime,
+            DictFieldData(
+                {'account_id': 'account_id', 'metadata': {'client_id': 'api_key', 'client_secret': 'api_secret'}}
+            ),
+            scope_ids=mock.Mock(spec=[])
+        )
+
+        # Mocked objects is a dict containing info about mocked entities.
+        # Try to be consistent giving key names, that will prevent from same mocks to be restored more than once.
```

Comment:
That was my first thought but no other way we can make set() of it and avoid mocking same events